### PR TITLE
feat(wizard): create wizard header

### PR DESCRIPTION
**Pull request by eweseong**
image

Create the component for wizard header section, it accepts 2 properties: `heading` and `actions`, see [file](https://github.com/eweseong/react-components/pull/2/files#diff-bfe74d50e6b3a4a271859e13706f9f7f6dcb617ee420339a70588ba15a826f4d) for more information.

### CODE REVIEW

File: lib/src/Wizard/Wizard.tsx [hunk 1]
```diff
@@ -66,83 +69,73 @@ export type WizardControlType = "next" | "prev" | "cancel" | "save" | "close";
 export type WizardStepState = "success" | "warning" | "danger" | "info";
 
 export const Wizard: React.FC<WizardProps> = React.memo(
-    React.forwardRef(({ heading, actions, steps, ...props }: WizardProps, ref: React.ForwardedRef<HTMLDivElement>) => {
+    React.forwardRef(({ heading, actions, steps, toggle }: WizardProps, ref: React.ForwardedRef<HTMLDivElement>) => {
```
Comment (kherP) [inline lib/src/Wizard/Wizard.tsx:72]: are the other props temporarily removed or this is how it going to be?

Comment (eweseong) [reply lib/src/Wizard/Wizard.tsx:72]: these are the current props that i could identify, do you think the `...props` should be retained?

Comment (kherP) [reply lib/src/Wizard/Wizard.tsx:72]: if we want to follow the standards we made for other components, we actually allow user to pass the props for the wrapper element to the component. In this case, `div` is the wrapper element, so the developer can pass onClick or any other native props of div when defining this component.

Comment (eweseong) [reply lib/src/Wizard/Wizard.tsx:72]: will make the necessary changes

Comment (eweseong) [reply lib/src/Wizard/Wizard.tsx]: added the suggested changes :)

**Comment by eweseong** [comment]
moving wizard component into its own repository as there's multiple dependencies that is only used by wizard component, hence it make more sense to have its own repository

**Pull request closed without merging**